examples/gallery/plot/double-y-axes.py

```diff
@@ -0,0 +1,63 @@
+"""
+Double Y-axes
+-------------
+
+The ``frame`` parameter of the plotting methods of the :class:`pygmt.Figure`
+class can control which axes should be plotted and optionally show annotations
+tick marks, and grid-lines. By default, all 4 axes are plotted, along with annotations
+and tick marks (denoted **W**, **S**, **E**, **N**). Lower case versions (**w**, **s**,
+**e**, **n**) can be used to denote to only plot the axes with tick marks. We
+can also only plot the axes without annotations and tick marks using **l**
+(left axis), **r** (right axis), **t** (top axis), **b** (bottom axis).
+When ``frame`` is used to change the frame settings, any axes
+that are not defined using one of these three options are not
+drawn.
+"""
+
+import numpy as np
+import pygmt
+
+# Generate common x values and two kinds of y values
+x = np.linspace(1.0, 9.0, num=9)
+y1 = x
+y2 = x ** 2 + 110
+
+fig = pygmt.Figure()
+
+# Plot y1
+# The bottom axis (S) is plotted with annotations and tick marks
+# The top axis (t) is plotted without annotations and tick marks
+# The left and right axes are not drawn
+fig.basemap(region=[0, 10, 0, 10], projection="X15c/15c", frame=["St", "xaf+lx"])
+# The left axis (W) is plotted with custom annotation spacing, tick spacing, and a label
+with pygmt.config(
+    MAP_FRAME_PEN="1.5p,blue",
+    MAP_TICK_PEN="0.5p,blue",
+    FONT_ANNOT_PRIMARY="12p,Helvetica,blue",
+    FONT_LABEL="16p,Helvetica,blue",
```

Comment:
Only show how to change the colors.
```suggestion
    MAP_FRAME_PEN="blue",
    MAP_TICK_PEN="blue",
    FONT_ANNOT_PRIMARY="blue",
    FONT_LABEL="blue",
```